### PR TITLE
Blog about setting up of rails

### PR DESCRIPTION
**Pull request by mingsheng88**
3 of the setup mistakes that I found so far

### CODE REVIEW

File: _posts/2017-01-30-getting-your-rails-setup-right [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+layout: post
+title:  "Getting It Right - Rails Setup"
+date:   2017-01-30 11:39 AM
+categories: ruby-on-rails
+author: Mingsheng Gan
+---
+
+Setting up your Rails project correctly is of utmost importance.  Obvious setup instructions somehow gets missed, and entire teams suffer from the repercussions of a bad setup.
+
+This post describes some common of the common configuration mistakes specific to the `test-suite` that we have rectified. Simple but essential, we hope that this article helps you improve your codebase and enhance your developmental workflow.
```
Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right]: `some common configuration mistakes`

**Comment by MacTynow** [comment]
@mingsheng88 I renamed your file so that it's served by Jekyll properly.

**Comment by MacTynow** [comment]
@mingsheng88 You need to fix your reference links at the end, currently they show up incorrectly.

File: _posts/2017-01-30-getting-your-rails-setup-right.markdown [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "Getting It Right - Rails Setup"
+date:   2017-01-30 11:39 AM
+categories: ruby-on-rails
+author: Mingsheng Gan
+---
+
+Setting up your Rails project correctly is of utmost importance.  Obvious setup instructions somehow gets missed, and entire teams suffer from the repercussions of a bad setup.
```
Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: 2  spaces after `.` 😄

File: _posts/2017-01-30-getting-your-rails-setup-right.markdown [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+This post describes some common common configuration mistakes specific to the `test-suite` that we have rectified. Simple but essential, we hope that this article helps you improve your codebase and enhance your developmental workflow.
```
Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: common x2

**Comment by mingsheng88** [comment]
@MacTynow once there's nothing else, then I think its good to go

File: _posts/2017-01-30-getting-your-rails-setup-right.markdown [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+author: Mingsheng Gan
+---
+
+Setting up your Rails project correctly is of utmost importance. Obvious setup instructions somehow gets missed, and entire teams suffer from the repercussions of a bad setup.
+
+This post describes some common configuration mistakes specific to the `test-suite` that we have rectified. Simple but essential, we hope that this article helps you improve your codebase and enhance your developmental workflow.
+
+## 3 Common Setup Mistakes
+
+Learning to observe and having zero tolerance for obvious repetitions go a long way. Most popular gems typically consist of a configuration to simplify its usage. That, we should always keep in mind.
```
Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: Might want to rephrase this somehow, sounds a bit unorganized 🤔

File: _posts/2017-01-30-getting-your-rails-setup-right.markdown [hunk 5]
```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Getting It Right - Rails Setup"
+date:   2017-01-30 11:39 AM
+categories: ruby-on-rails
+author: Mingsheng Gan
+---
+
+Setting up your Rails project correctly is of utmost importance. Obvious setup instructions somehow gets missed, and entire teams suffer from the repercussions of a bad setup.
```
Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: s/gets/get/

Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: s/entire teams/entire team/

Comment (MacTynow) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: Agreed with the first one, but the second one is correct as it is.

File: _posts/2017-01-30-getting-your-rails-setup-right.markdown (hunk 4 above)
Comment (mingsheng88) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: Just gonna chop it away

File: _posts/2017-01-30-getting-your-rails-setup-right.markdown (hunk 5 above)
Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: Okay, the reason why i would prefer `entire team suffers` is that it's stated `1 project` in the first sentence, so more likely you are talking about 1 team but not all teams.
Either way is fine with me tho.

File: _posts/2017-01-30-getting-your-rails-setup-right.markdown [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+# After: She's in the closet now
+clean_codebase = create(:codebase)
+better_codebase = create(:codebase, :with_traits)
+```
+
+Keywords such as `create`, `build` or `build_stubbed` are typically used in almost every test case. Measuring the number of lines of codes affected, a simple fix like the above affects `416 / 7413` lines of code across `76 files` in the spec suite, improving 5% of the codebase with just 1 line of code change.
+
+### Symptom: RSpec required specifically in every spec file
+
+Do you need to require rails_helper in every file? Or, type `RSpec` before every `describe`?
```
Comment (lenguyenthedat) [inline _posts/2017-01-30-getting-your-rails-setup-right.markdown]: maybe:
"Do you need to `require 'rails_helper'` in every file?"

**Comment by lenguyenthedat** [comment]
`simple` and `simplify` are used 9 times in this article. You might wanna reduce them or use some synonyms where possible.

**Pull request merged**